frontend/benefit/applicant/public/locales/en/common.json

```diff
@@ -418,94 +418,94 @@
     },
     "decision": {
       "description": {
-        "accepted": "Helsinki benefit has been granted from {{dateRangeStart}} to {{dateRangeEnd}}",
-        "rejected": "No Helsinki benefit was granted"
+        "accepted": "The application has been approved and the Helsinki benefit has been granted for the period of {{dateRangeStart}} - {{dateRangeEnd}}",
+        "rejected": "The application was rejected. No Helsinki benefit is granted."
       },
       "headings": {
         "mainHeading": "Decision details",
-        "caseId": "Case identifier",
+        "caseId": "Diary number",
         "status": "Decision",
-        "decisionDate": "Date of decision",
+        "decisionDate": "Decision made on",
         "benefitPeriod": "Benefit period",
-        "benefitAmount": "Total amount of benefit",
-        "existingAlterations": "Alterations in employment"
+        "benefitAmount": "Total benefit granted",
+        "existingAlterations": "Change to employment"
       },
       "alterationList": {
-        "count_one": "{{count}} alteration in employment reported",
-        "count_other": "{{count}} alterations in employment reported",
-        "empty": "No reported alterations in employment. The employment has proceeded as expected."
+        "count_one": "{{count}} change in employment reported",
+        "count_other": "{{count}} changes in employment reported",
+        "empty": "No reported breaks in employment. Employment realised according to plan."
       },
       "actions": {
-        "showDecision": "View the decision",
-        "reportAlteration": "Report an alteration"
+        "showDecision": "View decision",
+        "reportAlteration": "Report changes to employment"
       }
     },
     "alteration": {
-      "title": "Report an alteration",
-      "explanation": "Let us know if the employee's work contract ends early or is suspended temporarily. You may need to pay some of the granted Helsinki benefit back if the work contract was altered.",
+      "title": "Report changes to employment",
+      "explanation": "Report it if the employed person’s employment ends or is temporarily interrupted during the benefit period. We may recover some of the Helsinki benefit if the employment relationship changes during the benefit period.\nAll fields marked with * are mandatory.",
```

Comment:
Seems the translation was updated, but I meant that adding the required fields text here in the first place means it will now appear twice:

```jsx
              <p>{t('common:applications.alteration.explanation')}</p> {/* once at the end of this message */}
              <p>{t('common:applications.pageHeaders.guideText')}</p> {/* and again as its own element here */}
```

Thus it should not be required to add that sentence to this translation at all for a second time.

Reply:
Duplicate text removed.